### PR TITLE
Update zh-rTW translation

### PR DESCRIPTION
**Pull request by 0140454**
Update translation for payment link and ledger.

@Lafudoci Could you help me to review the translation?

### CODE REVIEW

File: app/src/main/res/values-zh-rTW/help.xml [hunk 1]
```diff
@@ -226,14 +226,14 @@
     ]]></string>
 
     <string name="help_uri"><![CDATA[
-        <h1>Using a payment link</h1>
-        <p>You have started monerujo with a payment link. In order to send funds, please do the following:</p>
+        <h1>你正在使用付款網址</h1>
```
Comment (Lafudoci) [inline app/src/main/res/values-zh-rTW/help.xml]: 由於這是說明頁面的段落標題，我建議譯為"使用付款網址"即可。

Comment (0140454) [reply app/src/main/res/values-zh-rTW/help.xml]: 謝謝建議，已經更新翻譯了。

**Review by Lafudoci — APPROVED** [review]
Translation reviewed. @0140454, nice work!

**Review by erciccione — APPROVED** [review]
Thank you :)

**Pull request merged**